public_html/wp-content/mu-plugins/blocks/assets/src/speakers/speakers-select.js

```diff
@@ -39,6 +43,8 @@ class SpeakersSelect extends Component {
 						name={ optionData.label }
 						size={ 24 }
 						url={ optionData.avatar }
+
+						// formatting is messed up when no gravatar exists for user
```

Comment:
This might be a problem with our custom AvatarImage component not handling default avatars, rather than something with this select component. I'll need to dig further to figure this out.